charybdis-macros/src/scylla/from_row.rs

```diff
@@ -36,6 +36,7 @@ pub(crate) fn from_row(struct_name: &syn::Ident, fields: &CharybdisFields) -> Im
     });
 
     let generated = quote! {
+        #[allow(clippy::unwrap_used, clippy::unwrap_in_result)]
```

Comment:
Thanks for the PR!

Is this line needed? I guess with the default clippy settings it does not complain on unwrap.

Note that this trait implementation is copied from the scylla driver itself  and in future scylla versions, they will update it, so we probably not gonna need it.

Reply:
This will be needed when it's enabled, ideally the unwrap should be removed. But I just suppressed it for now

Reply:
Seems that their [updated implementation](https://github.com/scylladb/scylla-rust-driver/blob/dce9e9fbdcee42d3bdec024776d4a4555207831b/scylla-macros/src/from_row.rs#L6) does not use unwrap, so we will copy-paste new implementation (or ask them to implement skip field feature to FromRow derive as well).

Reply:
Sure, Any input needed from my side?

Reply:
No, I will release it after I copy-paste the new `FromRow` code.

Reply:
Sure, Then can we merge this